memgpt/__init__.py

```diff
@@ -1 +1,3 @@
 __version__ = "0.2.10"
+
+from memgpt.client.client import Client as MemGPT
```

Comment:
imo might make sense to mimic this style for dev familiarity: https://github.com/openai/openai-python